lib/speak.js

```diff
@@ -1,8 +1,10 @@
 const say = require("say");
 const config = require("./config");
+const { logger } = require("./logger");
 
 module.exports = (message) => {
   if (config["TD_SPEAK"]) {
+    logger.debug(`${__filename}: (${JSON.stringify(message)})`);
```

Comment:
I don't think we'd want this conditional based on `TD_SPEAK`. See if https://github.com/testdriverai/testdriverai/blob/14a934d118cc864c1fc80239b6384f8ef4558b0a/lib/commander.js#L34 isn't what you want already

Reply:
```suggestion
    logger.debug(`${__filename}: %j`, message);
```

Reply:
My thinking was that, since the whole block is gated by `config["TD_SPEAK"]`, if `TD_SPEAK` is disabled, then we shouldn't log messages that aren't being sent to `say`, right?

Reply:
`say` is literal text to speech, so I don't associate it with logging. are you coming from a POV of privacy?